index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3167ce6dab.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="header container">
+            <h1 class="header-heading">My logo</h1>
+            <nav>
+                <i class="header-icon fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+    <section class="section-info">
+        <div class="info-display container">
+            <h2 class="info-heading">I´m Maury <br> Glad to see you!</h2>
+            <p class="info-paragraph">I´m a software developer! I can help you build a project, feature or website Look
+                trough some of my work and experience! if you like what you see and have a project you need to coded,
+                don´t hestiate me to contact me</p>
+            <h3 class="info-heading2">Let´s Connect</h3>
+            <div class="info-icon">
+                <a href="https://twitter.com/MauryRodrguez6"><i class="fa-brands fa-twitter"></i></a>
+                <a href="https://www.linkedin.com/in/mauricio-gallegos-rodr%C3%ADguez-380a96183/"><i
+                        class="fa-brands fa-linkedin"></i></a>
+                <a href="mailto:trypticst@gmail.com?subject=HTMLlink"><i class="fa-duotone fa-m"></i></a>
+                <a href="https://github.com/maury18"><i class="fa-brands fa-github"></i></a>
+            </div>
```

Comment:
- [ ] Icons are looking good! It would be better to use `ul` and `li` tags to list them as this is a better practice for listing in HTML.